test/test_reads.py

```diff
@@ -819,6 +820,99 @@ def testTranslationOfStopCodonUAA(self):
             next(read.translations()))
 
 
+class TestDNAKozakRead(TestCase):
+    """
+    Test the DNAKozakRead class.
+    """
+    def testSequence(self):
+        """
+        A DNAKozakRead instance must have the correct sequence.
+        """
+        originalRead = DNARead('id', 'AAGTAAGGGCTGTGA')
+        read = DNAKozakRead(originalRead, 0, 4, 100.0)
+        self.assertEqual('AAGT', read.sequence)
+
+    def testStart(self):
+        """
+        A DNAKozakRead instance must store the correct start offset.
+        """
+        originalRead = DNARead('id', 'AAGTAAGGGCTGTGA')
+        read = DNAKozakRead(originalRead, 2, 4, 100.0)
+        self.assertEqual(2, read.start)
+        read = DNAKozakRead(originalRead, 4, 10, 100.0)
+        self.assertEqual(4, read.start)
+
+    def testStop(self):
+        """
+        A DNAKozakRead instance must store the correct stop offset.
+        """
+        originalRead = DNARead('id', 'AAGTAAGGGCTGTGA')
+        read = DNAKozakRead(originalRead, 2, 4, 100.0)
+        self.assertEqual(4, read.stop)
+        read = DNAKozakRead(originalRead, 4, 10, 100.0)
+        self.assertEqual(10, read.stop)
+
+    def testStartNegative(self):
+        """
+        A DNAKozakRead start offset must not be less than zero.
+        """
+        originalRead = DNARead('id', 'AAGTAAGGGCTGTGA')
+        error = 'start offset \\(-1\\) less than zero'
```

Comment:
It's a regex test, so you could use `error = r'^start offset \(-1\) less than zero$'` (and note the raw string (with the `r` prefix) that stops you from having to double the backslashes - I think that's right. Try it.)

Reply:
Ah yes! Works :)